packages/mattermost-redux/src/selectors/entities/general.ts

```diff
@@ -21,7 +21,7 @@ export function getFeatureFlagValue(state: GlobalState, key: keyof FeatureFlags)
     return getConfig(state)?.[`FeatureFlag${key}` as keyof Partial<ClientConfig>];
 }
 
-export function getLicense(state: GlobalState): any {
```

Comment:
Wondering if this change may affect any other calling code for this

Reply:
If everything still type checks, no. It won't change any run time code; any breakages seen would be at compile time.

Reply:
Sound good, thanks for clarifying!